index.html

```diff
@@ -164,6 +164,22 @@ <h2 class="card-title">
         </div>
     </div>
         </section>
+        <section class="contact-me display-flex flex-column align-middle">
+            <h1 class="section-heading neutrals txt-center">
+                I'm always interested in hearing about new projects, so if you'd like to chat please get in touch. 
+            </h1>
```

Comment:
- [ ] Please kindly use the `footer` tag for the contact sections since it is recommended to use in the project requirement and for semantic reasons.
- [ ] I think since `h1` is already used for the heading "I'm Marine", I suggest you should use other headings for this heading.